examples/experimental/dagster-airlift/dagster_airlift/test/airflow_test_instance.py

```diff
@@ -127,5 +136,44 @@ def make_dag_run(dag_id: str, run_id: str, start_date: datetime, end_date: datet
             "state": "success",
             "start_date": AirflowInstance.airflow_date_from_datetime(start_date),
             "end_date": AirflowInstance.airflow_date_from_datetime(end_date),
+            "run_type": "manual",
+            "note": "dummy note",
+            "conf": {},
         },
     )
+
+
+def make_instance(
+    dag_and_task_structure: Dict[str, List[str]],
+    dag_runs: List[DagRun] = [],
+) -> AirflowTestInstance:
+    """Return a single dag instance with a set of tasks. No runs or task instances are included."""
```

Comment:
This docblock is a little misleading. It creates task instances and it takes passed in runs

Reply:
Ah whoops. Docblock from a previous iteration.